### PR TITLE
Fix build of everything in benchmark folder

### PR DESCRIPTION
**Pull request by luckychess**
### Description of the Change
Benchmarks can be built again.

### Benefits

Now they can be used as intended. 

### Possible Drawbacks 
None.

### Usage Examples or Tests 
```
cmake -DBENCHMARKING=ON
make benchmark_example
make bm_proto_creation
make bm_query
make bm_pipeline
```

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -141,6 +141,7 @@ node ('master') {
   testing = true
   testList = '(module)'
   fuzzing = true // testing = true
+  benchmarking = true // testing = true
```
Comment (MBoldyrev) [inline Jenkinsfile]:
```suggestion
  benchmarking = testing
```

Comment (luckychess) [reply Jenkinsfile]: Technically benchmarking could be off while testing is on. I guess it's unnecessary to make benchmarking always on if testing is on.
I decided to leave a commentary here to notice that benchmarking will be built only if testing is on.

File: Jenkinsfile [hunk 2]
```diff
@@ -289,7 +290,7 @@ node ('master') {
   if(!x64linux_compiler_list.isEmpty()){
     x64LinuxBuildSteps = [{x64LinuxBuildScript.buildSteps(
       parallelism==0 ?x64LinuxWorker.cpusAvailable : parallelism, x64linux_compiler_list, build_type, build_shared_libs, specialBranch, coverage,
-      testing, testList, cppcheck, sonar, codestyle, doxygen, packageBuild, sanitize, fuzzing, coredumps, useBTF, use_libursa, forceDockerDevelopBuild, environmentList)}]
+      testing, testList, cppcheck, sonar, codestyle, doxygen, packageBuild, sanitize, fuzzing, benchmarking, coredumps, useBTF, use_libursa, forceDockerDevelopBuild, environmentList)}]
     //If "master" also run Release build
     if(specialBranch && build_type == 'Debug'){
       x64LinuxBuildSteps += [{x64LinuxBuildScript.buildSteps(
```
Comment (BulatSaif) [inline Jenkinsfile]: add  `benchmarking` to this function call

File: Jenkinsfile [hunk 3]
```diff
@@ -317,7 +318,7 @@ node ('master') {
   def x64MacBuildPostSteps = new Builder.PostSteps()
   if(!mac_compiler_list.isEmpty()){
     x64MacBuildSteps = [{x64BuildScript.buildSteps(parallelism==0 ?x64MacWorker.cpusAvailable : parallelism,
-      mac_compiler_list, build_type, coverage_mac, testing, testList, packageBuild, fuzzing, useBTF, environmentList)}]
+      mac_compiler_list, build_type, coverage_mac, testing, testList, packageBuild, fuzzing, benchmarking, useBTF, environmentList)}]
     //If "master" or "dev" also run Release build
     if(specialBranch && build_type == 'Debug'){
       x64MacBuildSteps += [{x64BuildScript.buildSteps(parallelism==0 ?x64MacWorker.cpusAvailable : parallelism,
```
Comment (BulatSaif) [inline Jenkinsfile]: add `benchmarking` to this function call

**Comment by luckychess** [comment]
Wow, 4 approvals!

**Pull request merged**